custom_components/homewhiz/translations/select.de.json

```diff
@@ -55,28 +55,11 @@
       "SPIN_1200": "1200",
       "SPIN_1400": "1400"
     },
-    "homewhiz__WASHER_PREWASH": {
-      "PREWASH_OFF": "Aus",
-      "PREWASH_ON": "Ein"
-    },
-    "homewhiz__WASHER_EXTRARINSE": {
-      "EXTRA_RINSE_OFF": "Nachsp\u00fclen aus",
-      "EXTRA_RINSE_ON": "Nachsp\u00fclen ein"
-    },
```

Comment:
I'm not sure why this is removed. No other entries in the translation files contain this information / translation.

Reply:
Replaced with HA switches. No need for translations any longer